package.json

```diff
@@ -5,22 +5,26 @@
   "main": "lib/main.js",
   "scripts": {
     "build": "tsc",
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "jest --silent",
+    "test-coverage": "jest --silent --coverage"
   },
   "author": "Shivam Gupta",
   "license": "MIT",
   "devDependencies": {
-    "@types/node": "^13.7.7",
-    "@types/js-yaml": "^3.12.1",
-    "typescript": "^3.8.3"
+    "@types/jest": "25.2.3",
+    "@types/js-yaml": "3.12.1",
+    "@types/node": "13.7.7",
+    "jest": "26.0.1",
+    "ts-jest": "26.0.0",
+    "typescript": "3.8.3"
```

Comment:
I see that you're bringing in only 3 new packages.. could you validate if all the newly introduced package references in package-lock file are relevant? 

one way is, remove that file and run an npm install locally

Reply:
Removed the package-lock.json and did `npm i`. Looks like only changes are because of removing the greater than sign in versions. I'll check in the new package-lock.json